fw/http.c

```diff
@@ -815,7 +813,7 @@ do { 								\
 		body_len = tfw_str_to_cstr(body, body_val, remaining);
 	}
 
-	status_line_len = strlen(status_line);
+	status_line_len = sizeof(status_line);
```

Comment:
`sizeof` return size of pointer in this case.

Reply:
Yes I make this change automatically according our discussion(